test/functional/pdo_sqlsrv/pdo_connection_resiliency.phpt

```diff
@@ -7,7 +7,8 @@ killing the connection and then not freeing it. The latter case is the only one
 that should fail. Finally, execute two queries in two threads on a recovered
 non-MARS connection. This should fail too.
 --SKIPIF--
-<?php require('skipif_protocol_not_tcp.inc');
+<?php require('skipif_mid-refactor.inc');
+      require('skipif_protocol_not_tcp.inc');
       require('skipif_version_less_than_2k14.inc');  ?>
 --FILE--
```

Comment:
maybe put skipif_mid-refactor.inc to the end? because it skips sql server < 2016?

Reply:
skipif_mid-refactor doesn't skip any SQL Server unless AE is set, in which case it will skip versions < 2016. That's what we want isn't it?